test/address_prop.ml

```diff
@@ -0,0 +1,37 @@
+open Pollinate
+open QCheck2.Gen
+open Generator.Test
+
+let create =
+  QCheck2.Test.make ~count:1000
+    ~name:"create from string and port is successful" (pair string int)
```

Comment:
Which makes me think: should we crash of the address is not well formatted? (for example "address" should fail and "a.b.c.d.e" should also fail).
What do you think? Is it overkill?